data/insights/subscriptions/domain.yml

```diff
@@ -74,3 +91,18 @@ responses:
         * {active} active subscriptions
         * {expiring} subscriptions that are expiring soon
         * {expired} that have already expired
+  utter_ask_form_subs_product_usage_subs_product_type:
+    - text: "Which product's usage are you looking for?"
+      buttons:
+        - title: RHEL
+          payload: rhel
+        - title: OpenShift
+          payload: openshift
+  utter_subs_product_usage_rhel_page:
+    - text: "Check out the [RHEL's usage page here](https://console.redhat.com/subscriptions/usage/rhel)."
+  utter_subs_product_usage_openshift_page:
+    - text: "Check out the [OpenShift's usage page here](https://console.redhat.com/subscriptions/usage/openshift)."
```

Comment:
Also, the wording is a little weird here. Possessive nouns are used but a simple noun would do. Maybe:

[RHEL usage page here]
[OpenShift usage page here]